ocs_ci/ocs/resources/mcg.py

```diff
@@ -420,11 +420,12 @@ def _check_aws_credentials():
 
         return creds_request, aws_access_key_id, aws_access_key
 
-    def create_new_aws_connection(self, conn_name=None):
+    def create_new_aws_connection(self, cld_mgr, conn_name=None):
```

Comment:
I used it, that's how I noticed that it needs to be changed..

Reply:
@ebenahar - Ack. We're not using it - I think I originally implemented it to test the creation of new backingstores, but since then a lot has changed and it's no longer relevant.
However, now it's relevant once more because a connection is needed for the creation of namespace buckets. The change looks good. @romayalon - I think you can resolve this convo.